digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/service/linearasset/DynamicLinearAssetServiceSpec.scala

```diff
@@ -184,7 +184,7 @@ class DynamicLinearAssetServiceSpec extends FunSuite with Matchers {
       createdLimit.linkId should be (388562360)
       createdLimit.sideCode should be (SideCode.AgainstDigitizing.value)
       createdLimit.value should be (Some(propertyData3))
-      createdLimit.createdBy should be (Some("unittest"))
+      createdLimit.createdBy should be (Some("test"))
```

Comment:
The two asset types were being tested. The problem was that the names were not changed. The name of the `oldLimit `will be changed to `createdLimit1 `
```
 val createdLimit = ServiceWithDao.getPersistedAssetsByIds(typeId, Set(createdId(1))).head
 val oldLimit = ServiceWithDao.getPersistedAssetsByIds(typeId, Set(createdId.head)).head
```

And instead of getting the `createdId(1))` I will get the `createdId.head` and `createdId.last`